service/application/src/main/java/com/solace/maas/ep/event/management/agent/config/eventPortal/EventPortalProperties.java

```diff
@@ -24,6 +24,9 @@ public class EventPortalProperties {
     private Boolean managed = false;
     private String incomingRequestQueueName;
 
+    private int waitAckScanCompleteTimeout = 300;
+    private int waitAckScanCompletePollInterval = 10;
+
```

Comment:
I think we need to indicate that these are in seconds,  `waitAckScanCompleteTimeoutInSeconds` and `waitAckScanCompletePollIntervalInSeconds` ?

Reply:
Is 5 minutes enough? I think we should use at least 10 minutes to start with @moodiRealist  what do you think?